cli/statik/statik.go

```diff
@@ -6,7 +6,9 @@ import (
14\xc2\xe9\xcf	\x07M\x11\xc8\xa7\x1f\xf1s\xf4\x1aQ\xd0\x9c\xff\xe46C7\x91\xfb/\xf2\xee\xc3\xe9\xe9\xd9\xbbw\xddJHl\xe6zvP\xff\x14\xe5S\xd54X\xa7\xb3\x0d\x96\xa8\x19\xafO\xce\xdf\x9c\xbd\xeanW\xf6^sa\x9c?\x03y\xf1Q&\xcf\n\xa9\xbai\xb4\x03+\x90\x7f\xa2N8\x0e\xc4\xber\xf9\xe36\xb5\xb0\x1cT\x8b<\x05/\xe3\x86\xf5\xe2\x8d?\xc5v\x8f\xbb+\x82#q\x03\xc3T\x0c;\x9aL\x12\xf1\xa1\x08g\x92\xdf\x1d\xd3\x00\xfa\xfe\xfe\x07\xe1lr\xd0C\x89\xfcc\x170\xf6\x00c\xf7\xd3\x01|\x82\x99\xcc\x06\x12\xe1\xd4\xe7\xdd\xb0\xb2\x1b\xf6\xd0pe\x801\xaf2\x856\xdd1\x9a\xad\n\xa9\xea\x97\xbfF\x199^a\xaa\x96\xa0\xfe@s\x0c\xd31\xba\x0f^\x8f\x86J\xb1N\x8c\xedu\xdc\xfd\xb3\x17\xf8M\xf4\x90\x9c/A\xaa7\xa0a=X_\xa0\xbd:\x1f\xf9O=\xcd\xfa\xdc\x0d+\x0e\x98\x01Z\x0cq\xfd\x81\x80\x8c\xcc\xe0V\xa3\xe4\xe4\xf2\x1cG\x01T\x10@\xce)\x14\xfbF~\xfa>(\xf0,\x18\x9c.,J\x0e\x86/\x99\x8e\xe3\x89\x1f\xd6\x94\x00\xb2\xc2\x97\x9b\xe0\x1e\xb73><|\x16\xeb\xf9\x18\xaf\x85\x1f\xfa!\x0bq\x92\xc1\x8f.\xe1\xfeM.?\xbc'\xdc\x7fC#1\xfc\x9e\xe1r\xef\x83\xe8\"\xab\xeb\xdc\x87\xd8s\xc8\xaa\x92\x83?\xbb\xc82\xbac\xd2\x8d\xb3\x90\xee\x8bn\x18x\x08\xbf\xf0=\xf8\x1f$\xf9\x8f\x19v\xc9\x98|\x84z\xf9O/\xba\xba6e\xca\x03,+\xc8\xf5g7\xd0\xd4\xae\x93\x89\xfe\x85pD2\xf1\x9c\x87\x89y\x0fh\xc28(\x90\xf8\xb1\xeb3_\x9f\x15v_\x90\xae\xc0\xf3\xddO\x9f?\xbf\xe8.\xdf\x06\x82L\xc2\xe7\x03\x1c\xf74o^\xae,\xc5V\x93|z\x0diN\xaf|\xa6\x01\xbd'?\xfdt\xf6\xcbk\x98O\xd1\x8c\xc1ay \xdc\x94$\xd5\xe2\x1c\xaf\x9b\x95E\x01\x1f\x1cYs38 C\x81\x12\x1e\xe9\x98\xfe\xcb\x97\xa9\x91\xef\xac\xde(%\x11\x118\x01\x93W\x0f&\xf9\xdbF\xfc\xb9?\xd25\x18\xe4\xc5\\\xcb\x17'9\x0c\x1a\x80\x97\x84\xc3X\x89\xdd0\x80\x9fj\xe2?\xdd\x0e\x0cC\xde\xb8\x86\x03\x89\xec\x12\x07\x12\x94\"\xdc\xe5\xaa1\xc0\xed\xd8l'\x8cE\xe0\x07G8^\xad\xeb\xe0.\n\xae-\x1bg\xb4\x12\x04\x96\x042*\x9e\x8b\xf3\xf8\x1a)(=\x0f[\xfe\xc1\xd7fCT\xe4\xe0Y\x1c\xe9A\xb7\xe3\xda\xd4\"\xe1f@\xce\xfe}\xfe~\xc5\xa3\x97S\xaa\xd5\xe1\xfe\x1e\x90\x92\xffg\xcc`6\xcc\xe1\xeb\x90\xcb\xe2\x0eT[\x1d(\x90\xff\x05YL\xcb\xffC\xa1\x8d\xff\xbb\x1fH\xc8\xff\x0e\xca)\xcc\x8b\x18\xe0\xaf\x9a\xae\x13\xc3\xbc#\xd1\xac \x81@BB\x81\x0bv\x18\x95\x84\xf2\x10A\xa3\xa2\x03\x12\xc2\xe2F\xd5l\xc2Y\x18Z\xd4	X\xcd\x81\xbd\x00\xa9f\xea\xa7\x1c\x84\x0c\xc7D\x03\xdb\xab\x8bw\xc7\xcf\x84N\xc7\xf2\x9c\xb9\x1a\x14\xc0}\x0c\xa3\xf39\x8e\x8e\xc6D\xc2\xdd\xcb\xfc\x08\x98'\xbb\xc3Jl\x87\x8e%~4 (C\x98\xc0\xfa\xa2\xda\xb9\xe8\x0fl\x16\x17\xd8\xf58\xb4\x1c\x93\x83\xc3\xd3\x8bc?\xb2\x0bJ\x0c\x07\x84\xa3\xaa\x8a\xac\xb3\x1b\xda\x95Ot?\xaa\x129&\xd0\x9a\xf1\xb3\xa8](\x1d\xd0\x07\x87\xf4\x07~\xdc\x19\xd0P\xbe]#\x01\xc7\x84,_\xc8KHu\xff<T\xd9\xed\xa1\xe1\xc1 \x8a\xff\xfc\x0f!\xa6\x85\xa46j4\xc0\x85\x08@\x83\x82~\xd9de\xba\x8d\xd8x\xa0L,\xf7\x03\xff\xc46G\xb6\xf4\xc9\xa5))\x19\x93\x9b\xda\xe6\x82C2\x1d\x87\x03u\xdc;\\\x0eY\xec\xabB7|\x0c\x07<\xfc\n\xbf:\xbeRS~\x88ts\x06R\xe5\x8c\xe2\x05\xdc\xa2q\xda\xaa\xb7Y\xc4;\xefr@\xc7W\xa0\xce}\xdb\x05\xb9\xc12\xad/n\xdfL\xf1\xff\xe0\xcf\x86-@\x05\xfc_\x1c\xcai\xff\x7fI\xe2[\xfe\xbf\x17X\xc7\xffMbi\x16\x9bR\x0d\xe4\xd1\xab\
```

Comment:
Nit: Newline at end of file is missing.

Reply:
This is a generated file: https://github.com/SUSE/carrier/blob/main/cli/Makefile#L59

Reply:
`make tools` always pulls the latest statik afaict. So this diff may have been the result of an updated statik. Should we stub the statik version?

Reply:
Last commit of statik is from April, so not the case.

Reply:
I suspect that a `make fmt` may fix the issue.

Wrt latest statik ... For reproducible builds we really have to stub it, I believe.

Reply:
Missed this before I merged. I fixed it on `main` directly: https://github.com/SUSE/carrier/commit/6759a2cbbc9d74fda17d2ac20705b6a8d31ef8e4

I'll also have a look on how to stub the version.